src/exec/mod.rs

```diff
@@ -99,6 +99,8 @@ impl CommandTemplate {
                     "{/}" => tokens.push(Token::Basename),
                     "{//}" => tokens.push(Token::Parent),
                     "{/.}" => tokens.push(Token::BasenameNoExt),
+                    "{-}" => tokens.push(Token::StripPrefix),
+                    "{//-}" => tokens.push(Token::ParentStripPrefix),
```

Comment:
To be honest, I'd like to remove these new placeholders. I don't think there's a need for this and I don't want users to be overwhelmed when looking at the placeholder syntax.

Reply:
That seems reasonable to me. If there really is a need for them we can always add them later.